apps/condo/pages/billing/index.tsx

```diff
@@ -19,7 +17,7 @@ const BillingPage = () => {
 
     const userOrganization = useOrganization()
     const organizationId = get(userOrganization, ['organization', 'id'], '')
-    const canManageIntegrations = get(userOrganization, ['link', 'role', 'canManageIntegrations'], false)
+    const canManageIntegrations = get(userOrganization, ['link', 'role', 'canReadBillingReceipts'], false)
```

Comment:
We also have a ton of access checks in billing domain. Please check /access.js

Don't we need to change them too?

Reply:
Changed accesses in billing domain